site/static/examples/jobs/sample-pytorchjob.yaml

```diff
@@ -14,8 +14,10 @@ spec:
         spec:
           containers:
             - name: pytorch
-              image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
-              imagePullPolicy: Always
+              image: docker.io/kubeflowkatib/pytorch-mnist-gpu:latest
+#              if you don't have gpu, use pytorch-mnist-cpu to save download image time
+#              image: docker.io/kubeflowkatib/pytorch-mnist-cpu:latest
```

Comment:
```suggestion
              image: docker.io/kubeflowkatib/pytorch-mnist-cpu:latest
#              If you have gpu, pytorch-mnist-gpu would be helpful.
#              image: docker.io/kubeflowkatib/pytorch-mnist-gpu:latest
```

Could you use the cpu image? For just confirming kueue behavior, GPUs are not required.

Reply:
the CPU and GPU image test all is well on minikube cuda 12.4

Reply:
I meant that for just confirmation kueue behavior, we should not waste the volume and download time.

Reply:
how about use this image `docker.io/kubeflowkatib/pytorch-mnist-cpu:v1beta1-21320b6` ?

Reply:
SGTM
It is what I wanted to say.

Reply:
anything else to add before approval?

Reply:
how about update all the `imagePullPolicy`  to  `Always` in the `examples/jobs `direcotry